### DISCUSSION TITLE
What is the correct way to update applictions & monitor question.

### DISCUSSION THREAD

**Discussion by RWayne93** [prasanthrangan/hyprdots#868] [General]
for example if i want to update my vscode because a new release came out. What are the steps i would take to update it and keep all the settings and the dynamic theme changing that these dotfiles provide? 

i updated vscode once and lost all of the extensions and the dynamic theme changing and had to reapply the dot files doing ./install.sh is there an easier way to do this? 

And as far as monitors go here is my below configuration 
monitor=HDMI-A2,3440x1440,0x0,1
monitor=DP-3,3440x1440, 3440x0,1
monitor=eDP-1,1920x1080, 5360x0,1

for some reason the second monitor works however the wallpapers and waybar all disappear. They come back if i comment them out and switch back to auto...I know this is a hyprland thing but curious if anybody in this community has had issues with multi monitor setups and how did you fix them?

**Comment by kRHYME7** [comment]
Hi @RWayne93 
I will make a thread for restore/backup instructions etc for now see this thread 

Firstly you should run  ``` Hyprdots updates ``` to pull updates

then follow this to manage your files using a control file. 
https://github.com/prasanthrangan/hyprdots/discussions/863

or you can use ``` Hyprdots restore defaults ``` then you will get an interactive listing that lets you mark to PRESERVED, SYNC,OVERWRITE etc
